Builds/linux/README.md

```diff
@@ -12,6 +12,8 @@ dependencies using that distribution's package management tools.
 
 ## Dependencies
 
+gcc-7 or later is required.
```

Comment:
...or clang-5 (not sure if we want to mention that...)

Reply:
I'm not sure either.

Reply:
@HowardHinnant @mellery451 What do you think about updating line 9 to say something like: `Development is regularly done on Ubuntu 18.04 with gcc. Note: Ubuntu 16.04 users may need to update their compiler (see the dependencies section)`. 

As for the clang mention, I don't think very many people use clang on linux, but we may as well document that as well.

Reply:
I'm fine with rewording line 9 as you suggest as long as Mike is too (I'm not qualified to have an opinion on it).

The one thing that concerns me with documenting clang on linux is the implication that this is a supported configuration.  To the best of my knowledge our official support is limited to gcc.

Reply:
I'm fine with the 16.04 note, but I'd omit the first part that mentions 18.04 - I think any distro is fine as long as you have sufficient toolchain support. Skipping the clang mention makes sense -- I agree that we should just stick with gcc for production-like builds.

Reply:
Everyone happy with:

> Note: Ubuntu 16.04 users may need to update their compiler (see the dependencies section).

?

Reply:
@mellery451 I'm fine with leaving out the 18.04 mention.